File: src/app/model/Formation.ts
export type Formation = {
  id: string;
  title: string;
  description: string;
  location: string;
  date: Date;
  tags: string[];
  distance: number;
  prix: number;
  horairestart: Date;
  horaireend: Date;
  nombreParticipant: number;
  nombreParticipantMax: number;
}

File: src/main/java/be/ephec/pdw/springboot/formation/Formation.java
package be.ephec.pdw.springboot.formation;

import java.time.LocalDate;
import java.util.List;
import java.util.UUID;

public record Formation(UUID id,
                        String title,
                        String description,
                        String location,
                        LocalDate date,
                        List<String> tags) {
}
